### PR TITLE
language: klasa#473 Update Klasa for Internal Sharding

### PR DESCRIPTION
**Pull request by Pandraghon**
Based on https://github.com/dirigeants/klasa/pull/473

### CODE REVIEW

**Comment by Pandraghon** [comment]
Why didn't Github show me the last update ? :thinking:

**Comment by Pandraghon** [comment]
Should be good, @kyranet

File: languages/es-ES.js [hunk 1]
```diff
@@ -155,7 +155,7 @@ module.exports = class extends Language {
 				`• Klasa        :: v${klasaVersion}`,
 				`• Discord.js   :: v${discordVersion}`,
 				`• Node.js      :: ${processVersion}`,
-				this.client.options.shardCount ? `• Shard        :: ${((msg.guild ? msg.guild.shardID : msg.channel.shardID) || this.client.options.shardId) + 1} / ${this.client.options.shardCount}` : ''
+				this.client.options.shardCount ? `• Shard        :: ${(message.guild ? message.guild.shardID : 0) + 1} / ${this.client.options.totalShardCount}` : ''
```
Comment (kyranet) [inline languages/es-ES.js]: And same everywhere else:
```suggestion
				`• Shard        :: ${(message.guild ? message.guild.shardID : 0) + 1} / ${this.client.options.totalShardCount}`
```

**Comment by Pandraghon** [comment]
Are we good, now? :pray:

**Pull request merged**